### PR TITLE
Ul production samples cross sections

### PR DESCRIPTION
**Pull request by NTrevisani**
Main change:
- HT-binned DY cross-sections were not consistent among the three years --> fixed, following reference https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns

Other minor changes:
- Top cross-sections updated according to reference https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopRefXsec
- QCD cross-sections updated for some samples. Reference is GenXSAnalyzer. Since samples from different years may give sligthly (but compatible results), I used 2017 samples as reference
- Tri-boson, WZJJ, and other minor background samples updated to more recent values.

To check which cross-section to updated, I looked at samples for which our current (legacy) value is not the same as XSDB. 

Signal cross sections left untouched, since we directly take them from YR.

### CODE REVIEW

File: NanoGardener/python/framework/samples/samplesCrossSections2016.py [hunk 1]
```diff
@@ -34,204 +36,218 @@
 
 
 ## W+jets
-samples['WJetsToLNu']                	.extend( ['xsec=61526.7',	'kfact=1.00',		'ref=E'] )
+samples['WJetsToLNu']                   	.extend( ['xsec=61526.7',	'kfact=1.00',		'ref=E'] )
 samples['WJetsToLNu_ext2']                	.extend( ['xsec=61526.7',	'kfact=1.00',		'ref=E'] )
 samples['WJetsToLNu-LO']                	.extend( ['xsec=61526.7',	'kfact=1.00',		'ref=E'] )
 samples['WJetsToLNu-LO_ext2']                	.extend( ['xsec=61526.7',	'kfact=1.00',		'ref=E'] )
 
-samples['WJetsToLNu_HT70_100']          .extend( ['xsec=1353.0',       'kfact=1.0',           'ref=W'] )
```
Comment (lviliani) [inline NanoGardener/python/framework/samples/samplesCrossSections2016.py:42]: Do we know why this xsec in 2016 is larger than the other years?

**Comment by lviliani** [comment]
I think the cross-section of the W+jets HT binned samples do not currently match the ones in the xsec DB.
Are we still using the old values? Should be updated probably.

**Comment by NTrevisani** [comment]
I eventually checked one by one the samples and their cross-sections for the four eras. Now it should be ok, even if I found some typos and inconsistencies in the samples files (i.e., Summer20UL....).

For the moment, I am keeping the three samplescrosssections files separate for the three eras, but maybe it indeed makes sense to have just one samplesCrossSections_UL.py for all the eras.

Again, I haven't looked at signal samples.

**Comment by NTrevisani** [comment]
I added the samplesCrossSections_UL.py file. It should serve as a single XS file for the four eras.

I spotted some typos or inconsistencies in the samples names and I fixed them.

File: NanoGardener/python/framework/samples/samplesCrossSections_UL.py [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+#       U       https://twiki.cern.ch/twiki/pub/CMS/MonoHCombination/crossSection_ZpBaryonic_gq0p25.txt
+#	V	https://twiki.cern.ch/twiki/bin/viewauth/CMS/SameSignDilepton2016
+#       W       https://cms-gen-dev.cern.ch/xsdb/
+#       Z       http://cms.cern.ch/iCMS/analysisadmin/cadilines?line=SMP-18-006
+#       Y       https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageBSMAt13TeV
+#       A1      https://indico.cern.ch/event/673253/contributions/2756806/attachments/1541203/2416962/20171016_VJetsXsecsUpdate_PH-GEN.pdf 
+#	X	Unknown! - Cross section not yet there
+
+### W+jets
+samples['WJetsToLNu']                   	.extend( ['xsec=61526.7',	'kfact=1.00',		'ref=E'] )
```
Comment (lviliani) [inline NanoGardener/python/framework/samples/samplesCrossSections_UL.py:38]: It should be `Samples` and not `samples` if I'm not wrong.

Comment (NTrevisani) [reply NanoGardener/python/framework/samples/samplesCrossSections_UL.py:38]: In the master branch, the old samplesCrossSections files used `samples`:
https://github.com/latinos/LatinoAnalysis/blob/master/NanoGardener/python/framework/samples/samplesCrossSections2016.py

It is true that in the samples list files, we use `Samples`: 
https://github.com/latinos/LatinoAnalysis/blob/master/NanoGardener/python/framework/samples/Summer16_102X_nAODv7.py

I just kept the same convention.

Comment (lviliani) [reply NanoGardener/python/framework/samples/samplesCrossSections_UL.py:38]: Right, sorry for the misunderstanding... I think the issue is that there is a mixed usage of `Samples` and `samples` in this file.

File: NanoGardener/python/framework/samples/samplesCrossSections_UL.py [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+samples['GluGluToWWToTNMN']              	.extend( ['xsec=0.05152',	'kfact=1.000',		'ref=W'] )
+samples['GluGluToWWToTNTN']              	.extend( ['xsec=0.05152',	'kfact=1.000',		'ref=W'] )
+
+
+### WZ
+samples['WZ']			                .extend( ['xsec=47.130',	'kfact=1.000',		'ref=E'] )
+
+samples['WZTo3LNu']		                .extend( ['xsec=4.666',  	'kfact=1.000',		'ref=X'] ) # X = https://cms.cern.ch/iCMS/jsp/openfile.jsp?tp=draft&files=AN2019_156_v8.pdf
+Samples['WZTo3LNu_mllmin4p0']	                .extend( ['xsec=4.666',   	'kfact=1.000',		'ref=X'] ) # X = https://cms.cern.ch/iCMS/jsp/openfile.jsp?tp=draft&files=AN2019_156_v8.pdf BUT KEEPING INCLUSIVE VALUE!!! NEED TO CHECK!
+Samples['WZTo3LNu_mllmin0p1']	                .extend( ['xsec=4.666',   	'kfact=1.000',		'ref=X'] ) # X = https://cms.cern.ch/iCMS/jsp/openfile.jsp?tp=draft&files=AN2019_156_v8.pdf BUT KEEPING INCLUSIVE VALUE!!! NEED TO CHECK!
```
Comment (lviliani) [inline NanoGardener/python/framework/samples/samplesCrossSections_UL.py:123]: Here for example it's `Samples`

**Comment by NTrevisani** [comment]
You are right. I think those with `Samples` are the samples I added at last. I committed a new version, with coherently `samples` (no capital letters).

**Pull request merged**